### PR TITLE
docs(frontend):AgileMethodologies-206

### LINKED ISSUE #206: Improve Agile Methodologies section
- Add User Stories Description : 

https://www.atlassian.com/agile/project-management/user-stories

- Add Planning Session Description 
- Add Grooming session Description 
- Add Retrospective Session Description  
- Add Demo session description

### PR DESCRIPTION
**Pull request by steph222**
### GH Issue

Resolves #206  

### Steps to test
1. Go to Agile Methodologies.
2. Check content is right.

#### CheckList
- [ ] Add User Stories Description
- [ ] Add Planning Session Description
- [ ] Add Grooming session Description
- [ ] Add Retrospective Session Description
- [ ] Add Demo session description

### CODE REVIEW

File: docs/agile-methodologies.md [hunk 1]
```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+### How to write user stories
+- **Definition of Done:** The story is generally “done” when the user can complete the outlined task.
+- **Outline subtasks or tasks:** Decide which specific steps need to be completed and who is responsible for each of them.
+- **User personas:** For Whom? If there are multiple end users, consider making multiple stories.
+- **Ordered Steps:** Write a story for each step in a larger process.
+- **Listen to feedback:** Talk to your users and capture the problem or need in their words.
+- **Time:**  Since stories should be completable in one sprint, stories that might take weeks or months to complete should be broken up into smaller stories or should be considered their own epic.
+
+### Examples:
+>- As a **manager**, I want to **be able to understand my colleagues progress**, so I **can better report our sucess and failures**.
```
Comment (leisterfrancisco) [inline docs/agile-methodologies.md:51]: Grammar mistake
**sucess** -> **success**

File: docs/agile-methodologies.md [hunk 2]
```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/MiaZhJyYUj0" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How often should you hold retrospectives?
+
+Retrospectives can be held more frequently, including for minor releases, each sprint or even at daily or weekly standups.
+
+### **Demo Session**
+
+The sprint demo is invaluable for keeping stakeholders up to speed with the progress of product development. It allows them to feedback and discuss with the Product Owner and Scrum team any possible amendments to the Product Backlog which would help to maximise value.  
+He sprint demo shouldn’t take up too much of a Scrum team’s time. Ensuring that the sprint review meeting is an informal affair where questions, feedback and discussion are welcome – allows for prep time to be kept to a minimum.
```
Comment (leisterfrancisco) [inline docs/agile-methodologies.md:124]: Is **H**e or **Th**e at the beginning of, I'm not sure:
> He sprint demo shouldn’t take up too much of a Scrum team’s time

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/agile-methodologies.md (hunk 2 above)
Comment (steph222) [reply docs/agile-methodologies.md:124]: Is The, thanks!!

File: docs/agile-methodologies.md (hunk 1 above)
Comment (steph222) [reply docs/agile-methodologies.md:51]: Thanks!!

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

File: docs/agile-methodologies.md [hunk 3]
```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+- **Outline subtasks or tasks:** Decide which specific steps need to be completed and who is responsible for each of them.
+- **User personas:** For Whom? If there are multiple end users, consider making multiple stories.
+- **Ordered Steps:** Write a story for each step in a larger process.
+- **Listen to feedback:** Talk to your users and capture the problem or need in their words.
+- **Time:**  Since stories should be completable in one sprint, stories that might take weeks or months to complete should be broken up into smaller stories or should be considered their own epic.
+
+### Examples:
+>- As a **manager**, I want to **be able to understand my colleagues progress**, so I **can better report our success and failures**. 
+>- As a **customer** , I want **shopping cart feature** so that **I can easily purchase items online**.
+>- As an **manager**, I want to **generate a report** so that **I can understand which departments need more resources**.
```
Comment (Edgar097) [inline docs/agile-methodologies.md:53]: "As a manager"

Comment (Edgar097) [reply docs/agile-methodologies.md:53]: or it could be "As manager"

**Review by ldrojas — APPROVED** [review]
Great! Just check the images and videos loading correctly.

File: docs/agile-methodologies.md [hunk 4]
```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+1. Discuss any new information that may impact the plan.
+2. Confirm any currently known issues and concerns and record as appropriate.
+3. Review the definition of **Done** and make any appropriate updates based on technology, skill, or team member changes since the last sprint.
+4. Present proposed product backlog items to consider for the sprint backlog.
+5. Determine the needs, sign up for work, and estimate the work owned.
+6. Product Owner answers clarifying questions and elaborates acceptance criteria.
+7. Confirm any new issues and concerns raised during meeting and record.
+8. Confirm any assumptions or dependencies discovered during planning and record.
+9. ScrumMaster calls for a group consensus on the plan.
+10. Team and Product Owner signal if this is the best plan they can make given what they know right now.
```
Comment (Edgar097) [inline docs/agile-methodologies.md:85]: I think is better "point out" instead of "signal"

File: docs/agile-methodologies.md (hunk 3 above)
Comment (steph222) [reply docs/agile-methodologies.md:53]: Thanks!!

File: docs/agile-methodologies.md [hunk 5]
```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/MiaZhJyYUj0" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How often should you hold retrospectives?
+
+Retrospectives can be held more frequently, including for minor releases, each sprint or even at daily or weekly standups.
+
+### **Demo Session**
+
+The sprint demo is invaluable for keeping stakeholders up to speed with the progress of product development. It allows them to feedback and discuss with the Product Owner and Scrum team any possible amendments to the Product Backlog which would help to maximise value.
```
Comment (Edgar097) [inline docs/agile-methodologies.md:123]: "maximise" is correct but it is better to use "maximize"

File: docs/agile-methodologies.md (hunk 4 above)
Comment (steph222) [reply docs/agile-methodologies.md:85]: Oki, thanks!!

File: docs/agile-methodologies.md (hunk 5 above)
Comment (steph222) [reply docs/agile-methodologies.md:123]: Oki!!

**Review by Edgar097 — APPROVED** [review]

File: docs/agile-methodologies.md [hunk 6]
```diff
@@ -28,4 +28,107 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+## Agile project management
+
+### **User Stories**
+
+A user story is a lightweight method for quickly capturing the **who**, **what** and **why** of a product requirement. In simple terms, user stories are stated ideas of requirements that express what users need. User stories are brief, with each element often containing fewer than 10 or 15 words each.  
+The purpose of a user story is to articulate how a piece of work will deliver a particular value back to the customer. Note that **customers** don't have to be external end users in the traditional sense, they can also be internal customers or colleagues within your organization who depend on your team.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/apOvF9NVguA" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How to write user stories
```
Comment (leisterfrancisco) [inline docs/agile-methodologies.md:42]: Final question mark is missing

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/agile-methodologies.md (hunk 6 above)
Comment (steph222) [reply docs/agile-methodologies.md:42]: Thanks!!

**Comment by xavier506** [comment]
Lets add a short description of the standup meeting.

 FOR EXAMPLE

https://en.wikipedia.org/wiki/Stand-up_meeting

**Comment by steph222** [comment]
> Lets add a short description of the standup meeting.
> 
> FOR EXAMPLE
> 
> https://en.wikipedia.org/wiki/Stand-up_meeti

**Comment by xavier506** [comment]
@steph222  looks great! Let's add a section describing the Stand Up ceremony in Agile Methodologies

File: docs/agile-methodologies.md [hunk 7]
```diff
@@ -28,4 +29,137 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+### Benefits of daily Agile stand-ups
+
+- Daily stand-ups allow team members to work collaboratively toward project goals. 
+- Daily stand-up meetings are important for keeping Agile teams focused and on-task while providing quick, project-level updates to the rest of the team. 
+- The Agile methodology is all about versatility and flexibility, it’s important to make tweaks and improvements to your meetings to fit your team’s needs. 
+- Your daily stand-up should inform and draw out issues so that you can keep your project on track and get ahead of issues before they pop up.
+
+
+## Modern Agile
```
Comment (xavier506) [inline docs/agile-methodologies.md:155]: @steph222  lets move the modern agile section to the begining of the document

Comment (steph222) [reply docs/agile-methodologies.md:155]: Of course!

File: docs/agile-methodologies.md [hunk 8]
```diff
@@ -28,4 +29,137 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+### Demo steps
+
+1. **Tell a story:** This is one of the most important factors for a great demo, and also the most overlooked. Given the structured nature of Agile stories and epics, it’s easy to fall into the trap of simply enumerating the work that you’ve done. This isn’t necessarily bad, but it’s unlikely to excite non-technical stakeholders.
+2. **Let developers brag:** Whenever possible, it’s great to allow developers to present their own work, which helps to build confidence, morale, and a sense of ownership. A good compromise can be to have one organizing speaker with a different “guest” developer showing off their work each week.
+3. **Set expectations:** Setting expectations and providing context are critical for a successful demo. 
+4. **Action items:** List all completed stories in your agenda, weed out any stories that shouldn’t be demoed, organize the remaining stories roughly into scenarios or topics, decide whether to have developers help give parts of the demo and always set expectations and give context throughout the demo.
+
+### **Stand-up Session**
```
Comment (xavier506) [inline docs/agile-methodologies.md:137]: I would move the standup description before demo and retrospective descriptions

File: docs/agile-methodologies.md (hunk 7 above)
Comment (steph222) [reply docs/agile-methodologies.md:155]: Above the title  What are agile methodologies?

File: docs/agile-methodologies.md (hunk 8 above)
Comment (steph222) [reply docs/agile-methodologies.md:137]: Of course!!

File: docs/agile-methodologies.md [hunk 9]
```diff
@@ -28,4 +29,137 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
+
+True to its name, all participants in stand-ups usually remain standing to keep the meetings short and on-topic. However, digital stand-ups are also possible.
```
Comment (xavier506) [inline docs/agile-methodologies.md:141]: consider changing to : However, virtual stand-ups are...

File: docs/agile-methodologies.md [hunk 10]
```diff
@@ -28,4 +29,137 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+2. **Let developers brag:** Whenever possible, it’s great to allow developers to present their own work, which helps to build confidence, morale, and a sense of ownership. A good compromise can be to have one organizing speaker with a different “guest” developer showing off their work each week.
+3. **Set expectations:** Setting expectations and providing context are critical for a successful demo. 
+4. **Action items:** List all completed stories in your agenda, weed out any stories that shouldn’t be demoed, organize the remaining stories roughly into scenarios or topics, decide whether to have developers help give parts of the demo and always set expectations and give context throughout the demo.
+
+### **Stand-up Session**
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
```
Comment (xavier506) [inline docs/agile-methodologies.md]: Consider changing to : These meetings last approximately 15 minutes

File: docs/agile-methodologies.md [hunk 11]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+2. Creating new user stories in response to newly discovered needs.
+3. Re-assessing the relative priority of stories.
+4. Assigning estimates to stories which have yet to receive one
+correcting estimates in light of newly discovered information.
+5. Splitting user stories which are high priority.
+6. Get back to work.
+
+### **Stand-up Session**
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
```
Comment (xavier506) [inline docs/agile-methodologies.md:126]: consider changing to : These meetings last approximately 15 minutes ...

File: docs/agile-methodologies.md (hunk 10 above)
Comment (steph222) [reply docs/agile-methodologies.md]: Thanks!!

File: docs/agile-methodologies.md [hunk 12]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+A user story is a lightweight method for quickly capturing the **who**, **what** and **why** of a product requirement. In simple terms, user stories are stated ideas of requirements that express what users need. User stories are brief, with each element often containing fewer than 10 or 15 words each.  
+The purpose of a user story is to articulate how a piece of work will deliver a particular value back to the customer. Note that **customers** don't have to be external end users in the traditional sense, they can also be internal customers or colleagues within your organization who depend on your team.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/apOvF9NVguA" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How to write user stories?
+- **Definition of Done:** The story is generally “done” when the user can complete the outlined task.
+- **Outline subtasks or tasks:** Decide which specific steps need to be completed and who is responsible for each of them.
+- **User personas:** For Whom? If there are multiple end users, consider making multiple stories.
```
Comment (leisterfrancisco) [inline docs/agile-methodologies.md:59]: >  **User personas:**

Check this

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

File: docs/agile-methodologies.md [hunk 13]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+### Benefits of daily Agile stand-ups
+
+- Daily stand-ups allow team members to work collaboratively toward project goals. 
+- Daily stand-up meetings are important for keeping Agile teams focused and on-task while providing quick, project-level updates to the rest of the team. 
+- The Agile methodology is all about versatility and flexibility, it’s important to make tweaks and improvements to your meetings to fit your team’s needs. 
+- Your daily stand-up should inform and draw out issues so that you can keep your project on track and get ahead of issues before they pop up.
+
+
+### **Retrospective Session**
```
Comment (xavier506) [inline docs/agile-methodologies.md:142]: does using bold `**` tags  inside the  H3 tag `###` make a difference in styles?

File: docs/agile-methodologies.md [hunk 14]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+Is a meeting held after a product ships to discuss what happened during the product development and release process, with the goal of improving things in the future based on those learnings and conversations.  
+An agile retrospective forces the entire team to pause and reflect on what transpired and discuss what worked and what didn’t during a particular project.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/MiaZhJyYUj0" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How often should you hold retrospectives?
+
+Retrospectives can be held more frequently, including for minor releases, each sprint or even at daily or weekly standups.
+
+### **Demo Session**
```
Comment (xavier506) [inline docs/agile-methodologies.md:153]: Remove Bold

File: docs/agile-methodologies.md [hunk 15]
```diff
@@ -3,11 +3,25 @@ id: agile-methodologies
 title: Agile Methodologies
 sidebar_label: Agile Methodologies
 ---
+import useBaseUrl from '@docusaurus/useBaseUrl';
 
 ## What are agile methodologies?
 
 These are work methodologies that allow adaptation to the circumstances of the projects and reduce the documentation processes, they are generally used in projects that do not have a defined scope with certainty, which causes them to be very prone to change, because in these methodologies deliveries of functional products in a short time (2 to 4 weeks) allow the process of changes and correction of errors to be fast and not have a very high cost for the project in general.
 
+## Modern Agile
```
Comment (xavier506) [inline docs/agile-methodologies.md:12]: Lets use an H3 tag and change the title to  Modern Agility 

`### Modern Agility`

File: docs/agile-methodologies.md [hunk 16]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+## Agile project management
+
+### **User Stories**
+
+A user story is a lightweight method for quickly capturing the **who**, **what** and **why** of a product requirement. In simple terms, user stories are stated ideas of requirements that express what users need. User stories are brief, with each element often containing fewer than 10 or 15 words each.  
+The purpose of a user story is to articulate how a piece of work will deliver a particular value back to the customer. Note that **customers** don't have to be external end users in the traditional sense, they can also be internal customers or colleagues within your organization who depend on your team.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/apOvF9NVguA" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How to write user stories?
```
Comment (xavier506) [inline docs/agile-methodologies.md:56]: H4 `#### How to write user stories?`

File: docs/agile-methodologies.md [hunk 17]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+- **Definition of Done:** The story is generally “done” when the user can complete the outlined task.
+- **Outline subtasks or tasks:** Decide which specific steps need to be completed and who is responsible for each of them.
+- **User:** For Whom? If there are multiple end users, consider making multiple stories.
+- **Ordered Steps:** Write a story for each step in a larger process.
+- **Listen to feedback:** Talk to your users and capture the problem or need in their words.
+- **Time:**  Since stories should be completable in one sprint, stories that might take weeks or months to complete should be broken up into smaller stories or should be considered their own epic.
+
+### Examples:
```
Comment (xavier506) [inline docs/agile-methodologies.md:64]: H4

File: docs/agile-methodologies.md [hunk 18]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+### **Planning Session**
+
+Sprint planning is a timeboxed working session that lasts roughly 1 hour for every week of a sprint. 
+Is an event in the Scrum framework where the team determines the product backlog items they will work on during that sprint and discusses their initial plan for completing those product backlog items.  
+
+This agreement defines the sprint backlog and is based on the team’s velocity or capacity and the length of the sprint.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/2A9rkiIcnVI" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Who is involved?
```
Comment (xavier506) [inline docs/agile-methodologies.md:80]: H4

File: docs/agile-methodologies.md [hunk 19]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+- **Product Owner:** Identifies the candidate product backlog items and their relative priorities, as well as proposes a sprint goal.
+
+- **Team Members:** Determine how many of the product backlog items they forecast they will be able to complete and determine how they will deliver those product backlog items.
+
+- **Scrum Master:** Typically facilitates sprint planning in order to ensure that the discussion is effective and that there is agreement to the sprint goal and that the appropriate product backlog items are included in the sprint backlog.
+
+### Planning steps
```
Comment (xavier506) [inline docs/agile-methodologies.md:88]: H4

File: docs/agile-methodologies.md [hunk 20]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+10. Team and Product Owner point out if this is the best plan they can make given what they know right now.
+11. Get back to work.
+
+
+### **Grooming Session**
+Backlog grooming, also referred to as backlog refinement or story time, is a recurring event for agile product development teams. The primary purpose of a backlog grooming session is to ensure the next few sprints worth of user stories in the product backlog are prepared for sprint planning. Regular backlog grooming sessions also help ensure the right stories are prioritized.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/UpEBfS9SZGM" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Who is involved?
```
Comment (xavier506) [inline docs/agile-methodologies.md:108]: H4

File: docs/agile-methodologies.md [hunk 21]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+- **Product Owner:** Is tasked with facilitating backlog refinement sessions. However, that doesn’t mean they are solely responsible for holding backlog grooming sessions.
+- **Team Members:** These events are meant to be collaborative. That means the entire cross-functional team should be represented at refinement sessions.
+- **QA representatives:** You need the combined expertise of the various functions on your team to effectively flesh out your user stories.
+
+### Grooming steps
```
Comment (xavier506) [inline docs/agile-methodologies.md:114]: H4

File: docs/agile-methodologies.md [hunk 22]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+A daily stand-up meeting is an opportunity for the project team to discuss a project’s progress at a high level. These meetings last approximately 15 minutes and allow each contributor to report on their accomplishments since the last stand-up meeting.
+
+True to its name, all participants in stand-ups usually remain standing to keep the meetings short and on-topic. However, virtual stand-ups are also possible. 
+
+In Agile project management, daily stand-up meetings are essential. These meetings allow project members to share critical information, openly discuss issues, and hold themselves and each other accountable.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/YHMw3CL5cXE" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Benefits of daily Agile stand-ups
```
Comment (xavier506) [inline docs/agile-methodologies.md:134]: H4

File: docs/agile-methodologies.md [hunk 23]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+
+### **Retrospective Session** 
+
+Is a meeting held after a product ships to discuss what happened during the product development and release process, with the goal of improving things in the future based on those learnings and conversations.  
+An agile retrospective forces the entire team to pause and reflect on what transpired and discuss what worked and what didn’t during a particular project.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/MiaZhJyYUj0" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### How often should you hold retrospectives?
```
Comment (xavier506) [inline docs/agile-methodologies.md:149]: H4

File: docs/agile-methodologies.md [hunk 24]
```diff
@@ -28,4 +42,127 @@ It consists of the elaboration of a table or diagram in which three columns of t
...
+
+### **Demo Session**
+
+The sprint demo is invaluable for keeping stakeholders up to speed with the progress of product development. It allows them to feedback and discuss with the Product Owner and Scrum team any possible amendments to the Product Backlog which would help to maximize value.  
+The sprint demo shouldn’t take up too much of a Scrum team’s time. Ensuring that the sprint review meeting is an informal affair where questions, feedback and discussion are welcome – allows for prep time to be kept to a minimum.
+
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/Njxek3s3Shs" frameborder="0" allow="true" allowfullscreen></iframe>
+
+### Demo steps
```
Comment (xavier506) [inline docs/agile-methodologies.md:161]: H4

File: docs/agile-methodologies.md (hunk 13 above)
Comment (steph222) [reply docs/agile-methodologies.md:142]: This is the difference.
image

File: docs/agile-methodologies.md (hunk 14 above)
Comment (steph222) [reply docs/agile-methodologies.md:153]: Oki!!

**Review by xavier506 — APPROVED** [review]
LGTM

**Review by Edgar097 — APPROVED** [review]

**Review by leisterfrancisco — APPROVED** [review]
Good work 👍

**Pull request merged**